src/constants.js

```diff
@@ -0,0 +1,7 @@
+const EARLIEST_OFFSET = -2
```

Comment:
Ah, maybe this answers my question from before. Are these constants actually part of the protocol?

Reply:
👍